k8s/demo/common/ccd/ccd-demo-latest-chart.yaml

```diff
@@ -0,0 +1,74 @@
+apiVersion: flux.weave.works/v1beta1
+kind: HelmRelease
+metadata:
+  name: ccd-demo
+  namespace: ccd
+  annotations:
+    flux.weave.works/automated: 'true'
+    flux.weave.works/tag.java: 'glob:prod-*'
+spec:
+  releaseName: ccd-demo
+  timeout: 900
+  forceUpgrade: true
+  rollback:
+    enable: true
+    force: true
+  chart:
+    git: 'https://github.com/hmcts/chart-ccd'
+    ref: master
+    path: ccd
+  values:
+    ccd-api-gateway-web:
+      enabled: true
+      nodejs:
+        ingressHost: gateway-ccd-aks.service.core-compute-demo.internal
+    idam-pr:
+      releaseNameOverride: 'ccd-demo-idam-pr'
+      redirect_uris:
+        CCD:
+          - https://ccd-demo-case-management-web/oauth2redirect
+  
+          #CCD Front-end
+    ccd-case-management-web:
+      enabled: true
+      releaseNameOverride: ccd-demo-case-management-web
+      nodejs:
+        ingressHost: www-ccd-aks.service.core-compute-demo.internal
+    ccd-user-profile-api:
+      enabled: true
+
+    ccd-admin-web:
+      enabled: true
+  
+  #Dependencies
+    rpe-service-auth-provider:
+      enabled: true
+      java:
+        releaseNameOverride: ccd-demo-s2s
+
+    draft-store-service:
+      enabled: true
+      java:
+        ingressHost: ""
+
+    payment-api:
+      enabled: true
+      
+    dm-store:
+      enabled: true
+
+    ccd-case-activity-api:
+      enabled: true
+
+    ccd-case-print-service:
+      enabled: true
+
+  #Global values      
```

Comment:
global ?

Reply:
not global these need to be passed depends upon env

Reply:
You need global at this level , like : https://github.com/hmcts/cnp-flux-config/blob/master/k8s/aat/common/divorce/div-emca.yaml#L42

Reply:
oh, Thanks Praveen, added the tag